Updated requirements (#40)
Capstone-Skynet/Capstone-Skynet.github.io: docs/reqs/reqs.tex

    \item Lithium batteries cannot be larger than 100Wh for it to be carried onto a commerial airplane. For a 4S cell battery (14.8 nomial voltage), this capacity is 6700 mAh.
\end{enumerate}

\subsection{Environmental Considerations}
\begin{enumerate}[label=C.EN.\arabic*, wide=1cm, widest=3cm, leftmargin=*, font=\bfseries, noitemsep,topsep=0pt, parsep=4pt, partopsep=0pt]

>     \item A recycling plan must be included for environmentally impactful components (i.e. LiPo batteries).

Reviewer: **i.e.** is equivalent to "In other words"
**e.g.** is equivalent to "such as"

```suggestion
    \item A recycling plan must be included for environmentally impactful components (such as Li-Po batteries).
```
